src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RC2.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Cryptography;
+using System.ComponentModel;
+
+namespace System.Security.Cryptography
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class RC2 : SymmetricAlgorithm
+    {
+        protected int EffectiveKeySizeValue;
+
+        protected RC2()
+        {
+            LegalBlockSizesValue = s_legalBlockSizes;
+            LegalKeySizesValue = s_legalKeySizes;
+            KeySize = 128;
+            BlockSize = 64;
```

Comment:
I'm currently fixing the existing SymmetricAlgorithm classes to assign the fields instead of the properties.  Please switch these new ones to set KeySizeValue and BlockSizeValue.